cmd/root.go

```diff
@@ -74,6 +74,7 @@ func initConfig() {
 	viper.SetDefault("timeunit", "us")
 	viper.SetDefault("prometheus.fillprevious.period", "5 m")
 	viper.SetDefault("metrics.enabled", true)
+	viper.SetDefault("metrics.basicauth.enable", false)
```

Comment:
why not `metrics.basicauth.enabled` in order to be consistent with `metrics.enabled`